docs/Load_Cell.md

```diff
@@ -0,0 +1,102 @@
+# Load Cells
+
+This document describes Klipper's support for load cells. Basic load cell
+functionality can be used to read force data and to weigh things like filament.
```

Comment:
Consider adding a note that weighing things is only possible on printers with the load cells in the bed, and not those with the cell in the extruder.

Reply:
I don't want to get into applications because there are many. You don't even have to put it in the printer! Several people want to use this functionality to weigh filament spools mounted outside the printer. E.g. https://klipper.discourse.group/t/filament-spool-scale-hx711/19800/
- You could check before a print if there is enough filament on the spool based on the print weight
- Track filament used by weight.
- Track drying if the filament by weight lost
In the future this may be in an application that is just a dry box and not even a printer. 
One community member is even working on runout detection in a resin printer.